cavy.js

```diff
@@ -54,6 +55,7 @@ program
     'Swap the index files and start the report server without first building the app'
   )
   .option('-d, --dev', 'Keep report server alive until manually killed')
+  .option('--xml', 'Write out test results to an XML file')
```

Comment:
Let's tell the user what the filename will be, as it's hard-coded anyway.

```suggestion
  .option('--xml', 'Write out test results to cavy_results.xml')
```

(needs changing at the bottom too)